File: Api-Buen-Sabor/src/main/java/com/example/buensabor/entities/MercadoPagoDatos.java
package com.example.buensabor.entities;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Table;
import java.util.Date;

@Entity
@Data
@Table(name = "mercado_pago_datos")
public class MercadoPagoDatos extends EntityBean{

    private long identificadorPago;
    private Date fechaCreacion;
    private Date fechaAprobacion;
    private String formaPago;
    private String metodoPago;
    private String nroTarjeta;
    private String estado;
}

File: buen-sabor-empresa/src/app/models/mercado-pago-datos.ts
import { Pedido } from "./pedido";

export interface MercadoPagoDatos {
    
    identificadorPago: number;
    fechaCreacion: Date;
    fechaAprobacion: Date;
    formaPago: string;
    metodoPago: string;
    estado: string;
    pedido: Pedido;
}